Change naming of form questions

Aryan20 opened the pull request:
Form questions:

Change naming ->
1. “Choose one from a list” to “Radio button (Choose one option)” 
2. “Choose multiple from a list” to “Checkbox (Choose one or several options)”

This is done to better reflect what these options do.

fixes: fossasia/eventyay-tickets#994

![image](https://github.com/fossasia/eventyay-talk/assets/34372791/a2e9fa10-3314-4508-bbe8-a84882348736)


## Checklist

- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation.
- [ ] My change is listed in the ``doc/changelog.rst``.

Aryan20 commented:
Do I also need to update changelogs and docs?

mariobehling commented:
> Do I also need to update changelogs and docs?

yes, please

mariobehling reviewed (changes requested):
There are a lot of unrelated PO file changes in this PR that should not be there.

Aryan20 committed:
Change naming of form questions

Form questions:
1) Change naming “Choose one from a list” to “Radio button (Choose one option)”
2) “Choose multiple from a list” to “Checkbox (Choose one or several options)”

This is done to better reflect what these options do.

fixes: #12

Aryan20 commented:
Done, apologies for the delay

norbusan commented:
@mariobehling are you fine with the renaming? I am not sure which of those is actually better understandable.
Other than this, if we merge this change, we loose all the translations of this text, which by itself even worse.
So I am not convinced.

hpdang commented:
@Aryan20 do you want to solve the conflicts so we can merge this PR?

mariobehling commented:
Sorry, I missed this. @Aryan20 Actually the translations are not included. Do you think you could do this using AI?
